packages/fluent-bit/build

```diff
@@ -14,20 +14,7 @@ popd
 # Remove Fluent Bit's default config.
 rm -rf "$PKG_PATH/etc"
 
-# Add systemd unit files.
-# We use different unit files per role because Fluent Bit uses one service account on masters and another on agents.
-unit_template="/pkg/extra/dcos-fluent-bit.service"
-master_unit="$PKG_PATH/dcos.target.wants_master/dcos-fluent-bit.service"
-agent_unit="$PKG_PATH/dcos.target.wants_slave/dcos-fluent-bit.service"
-agent_public_unit="$PKG_PATH/dcos.target.wants_slave_public/dcos-fluent-bit.service"
-
-# Add the master unit file to the package.
-mkdir -p $(dirname "$master_unit")
-SERVICE="dcos-fluent-bit-master" envsubst '${SERVICE} ${PKG_PATH}' < "$unit_template" > "$master_unit"
-
-# Add the agent unit files to the package.
-# Agents and public agents have the same unit file contents.
-for unit in "$agent_unit" "$agent_public_unit"; do
-  mkdir -p $(dirname "$unit")
-  SERVICE="dcos-fluent-bit-agent" envsubst '${SERVICE} ${PKG_PATH}' < "$unit_template" > "$unit"
-done
+# Add systemd unit file.
+unit_file="$PKG_PATH/dcos.target.wants/dcos-fluent-bit.service"
+mkdir -p "$(dirname "$unit_file")"
```

Comment:
Do these nested quotes not cause a problem?

Reply:
It surprised me as well, but they do work as intended:

```
branden@crateria-14359 ~$ foo="/foo/bar"
branden@crateria-14359 ~$ echo "$(dirname "$foo")"
/foo
branden@crateria-14359 ~$
```

I'd normally remove them like I try to do with other surprising bash syntax, but these quotes will help make sure that bash always treats the enclosed value as a single string.

Reply:
bash works in mysterious ways. Thanks!